3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js

```diff
@@ -10,8 +10,15 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
+function checkDoubleDigits(num) {
   // TODO complete this function
+  return new Promise((resolve, reject) => {
+    if (num > 9 && num < 100) {
```

Comment:
Good, but my personal preference here would be to use an _inclusive_ range as it states more clearly what the edge cases of a double-digit number are.

```suggestion
    if (num >= 10 && num <= 99) {
```

Reply:
Sure I'll consider this next time